akka-stream-tck/src/test/scala/akka/stream/tck/HeadSinkSubscriberTest.scala

```diff
@@ -12,7 +12,7 @@ import scala.concurrent.Promise
 class HeadSinkSubscriberTest extends AkkaSubscriberBlackboxVerification[Int] {
   import HeadSink._
 
-  override def createSubscriber(): Subscriber[Int] = new HeadSinkSubscriber[Int]
+  override def createSubscriber(): Subscriber[Int] = new HeadOptionSinkSubscriber[Int]
```

Comment:
Technically this is no longer correct. You're testing HeadOptionSinkSubscriber rather than HeadSinkSubscriber